src/algorithms/calorimetry/ImagingTopoClusterConfig.h

```diff
@@ -7,14 +7,20 @@
 
 namespace eicrecon {
 
+  enum layerMode_enum {etaphi, xy};
```

Comment:
Hmmm... What's this additional enum for?

Reply:
it was a previous attempt to create an enum, and I forgot to remove that.

Reply:
Gotcha!

Reply:
removed

Reply:
Thanks!